packages/@aws-cdk/aws-iam/README.md

```diff
@@ -28,17 +28,17 @@ Managed policies can be attached using `xxx.addManagedPolicy(ManagedPolicy.fromA
 Many of the AWS CDK resources have `grant*` methods that allow you to grant other resources access to that resource. As an example, the following code gives a Lambda function write permissions (Put, Update, Delete) to a DynamoDB table.
 
 ```typescript
-const fn = new lambda.Function(...);
-const table = new dynamodb.Table(...);
+const fn = new lambda.Function(this, 'Function', functionProps);
+const table = new dynamodb.Table(this, 'Table', tableProps);
 
 table.grantWriteData(fn);
 ```
 
 The more generic `grant` method allows you to give specific permissions to a resource:
 
 ```typescript
-const fn = new lambda.Function(...);
-const table = new dynamodb.Table(...);
+const fn = new lambda.Function(this, 'Function', functionProps);
```

Comment:
Can we `void` the props out? Some form of `...` or `{ ... }` seems nicer there.

```suggestion
const fn = new lambda.Function(this, 'Function', (void '...', functionProps));
```

Reply:
@nija-at was hostile to this, as it makes the "source" example read awkwardly when people come across it while working on the source. And I tend to agree with him.